inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp

```diff
@@ -234,10 +252,23 @@ void MKLDNNPlugin::MKLDNNInferRequest::GetBlob(const char *name, InferenceEngine
             return;
         }
 
-        _outputs[name] = make_blob_with_precision(blobs[name]->getTensorDesc());
+        InferenceEngine::TensorDesc desc = blobs[name]->getTensorDesc();
+
+        auto outNodes = graph->GetOutputNodes();
```

Comment:
Code block below looks like WA. Please put a comment why we need it.

Reply:
done